CHANGELOG.asciidoc

```diff
@@ -3,6 +3,70 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.2.1]]
+=== Beats version 8.2.1
+https://github.com/elastic/beats/compare/v8.2.0...v8.2.1[View commits]
```

Comment:
```suggestion
https://github.com/elastic/beats/compare/v8.2.0\...v8.2.1[View commits]
```
As described in https://github.com/elastic/beats/pull/31537, the link is broken if you leave out the backslash.